### DISCUSSION TITLE
Welcome to Thino Discussions!

### DISCUSSION THREAD

**Discussion by Quorafind** [Quorafind/Obsidian-Thino#203] [Announcements 通告]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kubulo** [comment]
能否把热力图的颜色系，在黑暗模式下也做一个flomo版的色系供我使用，或者告诉我怎么修改` style.css `中哪些代码可以实现？谢谢你的贡献，非常Nice 的一款插件，可以安静的本地打开类Flomo在电脑上记东西了。
